browser/client/index.js

```diff
@@ -1 +0,0 @@
-console.log('hello world');
```

Comment:
Is there anything in this file now?

Reply:
Oh nvm I see you deleted it.  Our index should be _the_ place where the app is finally run from.  We can talk about how to use this later but I'd leave this around.

Reply:
yeah this is now under browser/src/index.js and contains:

```
import React from 'react';
import { createRoot } from 'react-dom/client';
import { App } from './App';

const container = document.getElementById('root');
const root = createRoot(container);
root.render(<App />);
```

Reply:
OK I now see what happened -- this was replaced with brwoser/src/index -- my bad.